### PR TITLE
#55 upload phrases endpoint

### PR DESCRIPTION
**Pull request by AlexKay28**
- add class connector to database
- add endpoint for possibility to upload data to databa

It's possible to upload data to base using approach showed below:

Screenshot_20211109_225214
se

### CODE REVIEW

File: backend/dbase/database_connector.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        - upload data to db
+        - load data from db
+        - update data in db
+
+    :param conn: database connection object
+    :param engine: connection to defined database
+    :param cur: cursor to db
+    """
+
+    def __init__(self, dbname, username, password, host, port):
```
Comment (alantishin) [inline backend/dbase/database_connector.py:18]: I think `def __init__(self, dbname: str, username: str, password: str, host: srr, port: int):`
looks much better then without types

File: backend/dbase/database_connector.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        """
+        Make sure the dbconnection gets closed
+        """
+        self.conn.close()
+        self.engine.dispose()
+        self.cur.close()
+
+    def upload_df_table(self, df, schema, table_name, if_exists="fail"):
```
Comment (alantishin) [inline backend/dbase/database_connector.py:51]: I think it`s better to separate connection from some actions with this connection.
One class is responsible for open/close connection
Other for working with it. 
S from SOLID

**Review by alantishin — CHANGES_REQUESTED** [review]

File: backend/api/upload_phrases.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import json
+import pandas as pd
+import traceback
+
+from . import api, request, jsonify, session
+
+
+@api.route("/upload_phrases", methods=["POST"])
```
[the remark this answers is not in the document]
Comment (AlexKay28) [reply backend/api/upload_phrases.py:8]: fixed

File: backend/dbase/database_connector.py (hunk 1 above)
Comment (AlexKay28) [reply backend/dbase/database_connector.py:18]: type hints are already added in #56

File: backend/dbase/database_connector.py (hunk 2 above)
Comment (AlexKay28) [reply backend/dbase/database_connector.py:51]: ok. method deleted from class, functionality implemented in session_object.py, in upload_phrases_to_db method

**Review by alantishin — APPROVED** [review]

**Pull request merged**